platform/8.0/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Jakarta Platform Spec 8.0"
+date: 2019-07-18T16:10:38-04:00 (use "date +%FT%T%z")
+summary: "Initial release of the Jakarta EE 8 Platform"
+---
+The Jakarta EE Platform defines a standard platform for hosting Jakarta EE applications.
+
+* [Jakarta EE Platform 8.0 Specification Document](./Platform-spec-8.0.pdf) (PDF)
+* [Jakarta EE Platform 8.0 Specification Document](./Platform-spec-8.0.html) (HTML)
+* Jakarta EE Platform 8.0 Javadoc - ??? (https://github.com/eclipse-ee4j/jakartaee-api)
+* Jakarta EE Platform 8.0 TCK - ???
+* Maven coordinates
+  * ???
+
+# Ballots
+
+## Release Review
+
+The Release Review Specification Committee Ballot concluded successfully on 2019-12-01 with the following results.
+
+|                       |  Yes    | No      | Abstain  |
+|-----------------------|---------|---------|----------|
+|Fujitsu                |         |         |          |
+|IBM                    |         |         |          |
+|Oracle                 |         |         |          |
+|Payara                 |         |         |          |
+|Red Hat                |         |         |          |
+|Tomitribe              |         |         |          |
+|EE4J PMC               |         |         |          |
+|Participant Members    |         |         |          |
+|Committer Members      |         |         |          |
+
+## Plan Review
+
+The Plan Review Specification Committee Ballot concluded successfully on 2019-09-31 with the following results.
+
+|                       |  Yes    | No  | Abstain  |
+|-----------------------|---------|-----|----------|
+|Fujitsu                |         |     |          |
+|IBM                    |         |     |          |
+|Oracle                 |         |     |          |
+|Payara                 |         |     |          |
+|Red Hat                |         |     |          |
+|Tomitribe              |         |     |          |
+|EE4J PMC               |         |     |          |
+|Participant Members    |         |     |          |
+|Committer Members      |         |     |          |
+
+# Compatible Implementations
+
+* Eclipse Glassfish 5.1
```

Comment:
Should probably link to an appropriate page.

Reply:
Good idea.  I wonder if the template should indicate that desire?